frontend/src/features/comparisons/inputs/CriteriaButtons.tsx

```diff
@@ -0,0 +1,249 @@
+import React, { useContext, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDrag } from '@use-gesture/react';
+import { Vector2 } from '@use-gesture/core/types';
+
+import { Box, Fade, IconButton, Slide } from '@mui/material';
+import { ArrowDropDown, ArrowDropUp } from '@mui/icons-material';
+
+import { useCurrentPoll } from 'src/hooks';
+import { TutorialContext } from 'src/features/comparisonSeries/TutorialContext';
+
+import { ComparisonRequest } from 'src/services/openapi';
+
+import CriterionButtons, { BUTTON_SCORE_MAX } from './CriterionButtons';
+import { useOrderedCriteria } from 'src/hooks/useOrderedCriteria';
+
+const SWIPE_TIMEOUT = 240;
```

Comment:
I feel the animation is a bit slow and frustrating when looking for a specific criterion. Why not using the same value as in the `EntitySelector`?
```suggestion
const SWIPE_TIMEOUT = 180;
```

Reply:
Addressed by https://github.com/tournesol-app/tournesol/pull/1968/commits/e3bed73721b6e8c20a710419c8174ff4ab3eafa6